src/data/brp.prepare.json

```diff
@@ -318,11 +318,131 @@
         "clear_schemas"
       ]
     },
+    {
+      "type": "execute_sql",
```

Comment:
We should still use create_table here (see previous discussion from last week in Slack)

Reply:
Done!